mungegithub/submit-queue/deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         - /mungegithub
         - --dry-run=true
         - --stderrthreshold=INFO
+        - --period=20s
```

Comment:
This is only true for mungers with the webhooks enabled, right? Do you want to make this an environment variable and set it to 20s for contrib and kubernetes?